op-batcher/batcher/service.go

```diff
@@ -206,6 +206,16 @@ func (bs *BatcherService) initChannelConfig(cfg *CLIConfig) error {
 		BatchType:          cfg.BatchType,
 	}
 
+	// override max frame size if set
+	if cfg.MaxFrameSize > 0 {
+		cc.MaxFrameSize = cfg.MaxFrameSize
+	}
+
+	// enable multi-frame txs if set
+	if cfg.MultiFrameTxs {
+		cc.MultiFrameTxs = true
+	}
+
 	switch cfg.DataAvailabilityType {
 	case flags.BlobsType:
```

Comment:
Just a note, there's a limit on `MaxFrameSize` set by plasma (after this switch block):

    if bs.UsePlasma && cc.MaxFrameSize > plasma.MaxInputSize
        return fmt.Errorf("max frame size %d exceeds plasma max input size %d", cc.MaxFrameSize, plasma.MaxInputSize)
    }

where default `MaxInputSize` is `130672`.

Reply:
To submit full blobs of `1.8MB` with this limit, we'll need ~15 frames per tx.

Reply:
Make sense, I'd not implement now, since we are using this setting just in rollup mode and the limit will disappear when [generic commitment](https://github.com/ethereum-optimism/optimism/issues/10400) will be there.